index.html

```diff
@@ -160,7 +164,7 @@ <h2>Tonic</h2>
                     </ul>
                 </div>
                 <div class="button">
-                    <button type="button" class="work-button">See Project</button>
+                    <button id="3" type="button" class="work-button">See Project</button>

```

Comment:
- [ ] kindly make the mobile version full height.
your version 👇🏾 
![Screenshot (185)](https://user-images.githubusercontent.com/65298048/175522551-2cfdb62a-98a6-493a-9bdb-b40eb6387a2a.png)
figma 👇🏾 
![Screenshot (186)](https://user-images.githubusercontent.com/65298048/175522583-30b31335-7487-483f-859f-f5b45cb93171.png)